libs/lb/lb-rs/libs/shared/src/crypto.rs

```diff
@@ -15,7 +15,7 @@ pub struct AESEncrypted<T: DeserializeOwned> {
     pub value: Vec<u8>,
     #[serde(with = "serde_bytes")]
     pub nonce: Vec<u8>,
-    #[serde(skip_serializing, default = "PhantomData::default")]
+    #[serde(skip_serializing)]
```

Comment:
I'm not sure why, but I started getting clippy failures here
```
error: use of `default` to create a unit struct
  --> libs/lb/lb-rs/libs/shared/src/crypto.rs:18:63
   |
18 |     #[serde(skip_serializing, default = "PhantomData::default")]
   |                                                               ^ help: remove this call to `default`
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#default_constructed_unit_structs
   = note: `-D clippy::default-constructed-unit-structs` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(clippy::default_constructed_unit_structs)]`
   ```
   https://github.com/lockbook/lockbook/actions/runs/8403428604/job/23013900257?pr=2582#step:4:459